Source/Shared/Cache.swift

```diff
@@ -27,8 +27,17 @@ public final class Cache<T: Cachable>: BasicHybridCache {
    - Parameter key: Unique key to identify the object in the cache
    - Parameter completion: Completion closure returns object or nil
    */
-
   public func object(_ key: String, completion: @escaping (_ object: T?) -> Void) {
     super.object(forKey: key, completion: completion)
   }
+  
+  /**
+   Tries to retrieve the cache entry from to the front and back cache storages.
+   
+   - Parameter key: Unique key to identify the cache entry in the cache
+   - Parameter completion: Completion closure returns cache entry or nil
+   */
+  public func cacheEntry(_ key: String, completion: @escaping (_ object: CacheEntry<T>?) -> Void) {
```

Comment:
@vadymmarkov yeah agreed! I wrestled with the name a bit. First I thought `objectEntry` but that means the same as `object`. And `objectMetadata` seemed too specific; felt like leaving a little wiggle room since it was an addition to the public interface. Maybe a better name might come to one of us in a dream, lol. Thanks for getting those changes in though; I'll be sure to open PRs/issues for anything I find as I continue to use this great library.